TCD1304.py

```diff
@@ -76,14 +73,17 @@ def master_clock (self):        # puts PWM with frequency of 2MHz on SQ1; master
     def sh_clock (self):           #  on SQ2 sets pulse for the integrationtime. Running the Sensor in shutter mode try with tint (min) = 10µs
         self.pwmgen.generate(["SQ2"], 1/integration_time, [0.5] ) 
 
+    def analog_signal_read (self, block:False):
+        self.scope.select_range('CH1', max_voltage_output)   # voltagerange should be fitted to the sensors output for better resolution. sensor otput is between 2V-3V due to datasheet
+        self.scope.configure_trigger(channel = 'CH1', voltage = min_voltage_output ) # starts recording , when voltage is over a certain level
+        self.scope.capture(channels=1, samples=integration_elements*samples_per_element, timegap=integration_time/samples_per_element, block=False)
+        self.icg_clock(self)
+        x, y= self.scope.fetch_data()
+        diff = abs(x,y[1, 0] - min_voltage_output )

```

Comment:
This doesn't look right. What are you trying to do, and what is the purpose of the `diff` variable?

Reply:
The diff varible was written in the example, I thought it would be necessary for the trigger, so I applied it to our method. If it's not, I'm going to take ot out.

Reply:
Nevermind, I found it: https://github.com/fossasia/pslab-python/blob/16d9860a97025443bbc30920b3a8eec536a35902/pslab/instrument/oscilloscope.py#L306
That example is indeed wrong, I need to fix it. Thanks for finding it!